lib/roo_on_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module RooOnRails
-  VERSION = '1.7.0'.freeze
+  VERSION = '1.8.0'.freeze
```

Comment:
We should do version/release separately I think is the general approach... @mezis ?